web/lib/queries/common/pageContentFields.ts

```diff
@@ -38,6 +38,22 @@ const pageContentFields = /* groq */ `
       ${downloadableImageFields},
     },
   },
+
+  _type == "textTeaser" => {
+    "type": _type,
+    "id": _key,
+    title,
+    "text": text[]{..., ${markDefs}},
+    "designOptions": {
+      "theme": coalesce(theme.value, 0),
```

Comment:
Again, not a fan of the 0, especially when reading it like this (imagine you have no context of the component, is reading the query, and sees that default value is 0, you get not value from it)